appservice/src/intTest/groovy/nu/yona/server/PinResetRequestTest.groovy

```diff
@@ -58,6 +58,42 @@ class PinResetRequestTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
+	def 'Request pin reset twice'()
+	{
+		given:
+		User richard = addRichard()
+		def firstResponse = appService.yonaServer.postJson(richard.pinResetRequestUrl, [:], ["Yona-Password" : richard.password, "Accept-Language" : "nl-NL"])
+		assert firstResponse.status == 200
+		sleepTillPinResetCodeIsGenerated(richard, firstResponse.responseData.delay)
```

Comment:
Maybe we could add a test with the situation that the PIN reset is requested before the delay interval...